### PR TITLE
CLN: Clean series/test_arithmetic.py

### PR DESCRIPTION
**Pull request by dsaxton**

### CODE REVIEW

**Review by rhshadrach — APPROVED** [review]
Nice cleanup, lgtm.

File: pandas/tests/series/test_arithmetic.py [hunk 1]
```diff
@@ -260,29 +260,26 @@ def test_sub_datetimelike_align(self):
...
-            tm.assert_series_equal(left.gt(right, axis=axis), left > right)
-            tm.assert_series_equal(left.ge(right, axis=axis), left >= right)
+        tm.assert_series_equal(left.eq(right, axis=axis), left == right)
+        tm.assert_series_equal(left.ne(right, axis=axis), left != right)
+        tm.assert_series_equal(left.le(right, axis=axis), left < right)
+        tm.assert_series_equal(left.lt(right, axis=axis), left <= right)
+        tm.assert_series_equal(left.gt(right, axis=axis), left > right)
+        tm.assert_series_equal(left.ge(right, axis=axis), left >= right)
+
+    @pytest.mark.parametrize("op", ["eq", "ne", "le", "le", "gt", "ge"])
```
Comment (jreback) [inline pandas/tests/series/test_arithmetic.py:275]: can you use the `all_compare_operators` fixture

File: pandas/tests/series/test_arithmetic.py [hunk 2]
```diff
@@ -260,29 +260,26 @@ def test_sub_datetimelike_align(self):
 
 
 class TestSeriesFlexComparison:
-    def test_comparison_flex_basic(self):
+    @pytest.mark.parametrize("axis", [0, None, "index"])
```
Comment (jreback) [inline pandas/tests/series/test_arithmetic.py:263]: can you use the `all_compare_operators` fixture

**Review by jreback — CHANGES_REQUESTED** [review]

**Review by jreback — APPROVED** [review]

**Comment by jreback** [comment]
thanks @dsaxton

**Pull request merged**